webpack.config.js

```diff
@@ -1,6 +1,20 @@
 var path = require('path')
 const CopyPlugin = require('copy-webpack-plugin')
 const entry = require('./package.json').main
+const fs = require('fs')
+const SvgoInstance = require('svgo')
+
+const svgo = new SvgoInstance()
+
+let iconName
+try {
+  iconName = JSON.parse(fs.readFileSync('manifest.konnector', 'utf8')).icon
+  // we run optimize only on SVG
+  if (!iconName.match(/\.svg$/)) iconName = null
+} catch (e) {
+  // console.error(`Unable to read the icon path from manifest: ${e}`)
+}
+const appIconRX = iconName && new RegExp(`[^/]*/${iconName}`)
```

Comment:
Wouldn't it be more logical to move the regex to l39?
l39 would become
`if(iconName && path.match(new RegExp(`[^/]*/${iconName}`)))`
and it will remove the need to have `appIconRX` const

Reply:
Indeed, it's done :)